### PR TITLE
Fix Replay In Console bug

### LINKED ISSUE #487: Replay in Console broken
In the firefox console:
```
ERROR errorHandler: TypeError: response.json is not a function
	 showHttpMessageDetails    https://zap//zapCallBackUrl/3312723844430812542?name=tools/history.js 92:2: {}
```

### PR DESCRIPTION
**Pull request by psiinon**
Fixes #487 

Adds a task for Replay in Browser - I thought this was easier for people to understand than Replay in Console, but can be solved by both.

WIP while I try to add some regression tests...

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

**Comment by psiinon** [comment]
OK, the regression tests are proving to be non trivial :(
I'll keep working on them but might as well get this merged as I dont know how long they will take.

File: src/main/zapHomeFiles/hudtutorial/en_GB/Resend.task.html [hunk 1]
```diff
@@ -0,0 +1,2 @@
+	<h2>Task</h2>
+	Select the GET request for this page in the History tab, change the User-Agent to include <strong>ZAP</strong> and 'Replay in Browser'<br>
```
Comment (thc202) [inline src/main/zapHomeFiles/hudtutorial/en_GB/Resend.task.html]: `and then 'Replay in Browser'`?

**Comment by thc202** [comment]
Worth update the changelog to mention the fix.

**Comment by psiinon** [comment]
Fixed the text and added a changelog line.

**Comment by thc202** [comment]
Thanks!

**Pull request merged**